### PR TITLE
Add mailing list form under link

### PR DESCRIPTION
**Pull request by Gmfholley**
I do not know if this is a working mailchimp form, but I copied it over from the splash page and styled it.

It seem to work locally, but the check will be that it actually adds to the mailing list.

Screen Shot 2019-12-11 at 1 23 47 PM
Screen Shot 2019-12-11 at 1 21 44 PM

### CODE REVIEW

File: _includes/components/mailing-list.njk [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              border: none;
+              border-radius: 1.5em;
+              font-size: 1.416666666667em;
+              display: inline-block;
+              margin: .5em auto;
+              text-decoration: none;
+              text-align: center;
+              white-space: nowrap;
+              height: auto;
+            }
```
Comment (Gmfholley) [inline _includes/components/mailing-list.njk:25]: Overrides to match our buttons

File: _includes/components/mailing-list.njk [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<div class="white-background">
```
Comment (Gmfholley) [inline _includes/components/mailing-list.njk:1]: The rest of this is copied over from the splash page site.

File: _includes/theme.css [hunk 3]
```diff
@@ -284,6 +284,12 @@ span.icon {
 		min-height: 3em; /* 72px /24 */
 	}
 }
+/* mailing list */
+.mailing-list {
+	margin: 0 auto;
+	max-width: 42rem;
+	margin-bottom: 2rem;
+}
```
Comment (Gmfholley) [inline _includes/theme.css:298]: I just added a tiny bit of styling to the surrounding div.

**Review by JenHomann — APPROVED** [review]
Looks good. I confirmed the sign-up did come through to Mailchimp.

**Pull request merged**